saleor/product/tasks.py

```diff
@@ -29,3 +34,14 @@ def update_variants_names(product_type_pk, saved_attributes_ids):
     instance = ProductType.objects.get(pk=product_type_pk)
     saved_attributes = Attribute.objects.filter(pk__in=saved_attributes_ids)
     return _update_variants_names(instance, saved_attributes)
+
+
+@app.task
+def update_all_products_minimal_variant_prices_task():
+    update_products_minimal_variant_prices()
+
+
+@app.task
+def update_products_minimal_variant_prices_of_discount_task(discount_pk):
```

Comment:
Are you gonna run these function somewhere?

Reply:
This is used when creating/editing discounts in Dashboard 1.0. @derenio What about support for updating the minimal variant price in API? That's should be our top priority.

Reply:
@maarcingebala I've added code to update the `minimal_variant_price` while creating/updating/deleting `Products`, `ProductVariants`, `Collections` and `Sales`.
We don't have to worry about `Categories` as these cascade delete products on removal. Also only products can be added to categories and we already handle that.